addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -88,7 +88,7 @@ def _make_request(self, url, params=False):
         proxy_error = response['result'].pop('proxy_error', False)
         if proxy_error:
             error_code = proxy_error['code']
-            if error_code == 'refresh_token_expired':
+            if error_code in ('refresh_token_expired', 'invalid_signature'):
```

Comment:
You don't check the signature when you request a new refresh token ?

Reply:
We do, but it gave problems when the refresh_token is not stored in the client db for one or the other reason.  I am going to close this anyways.  There are better ways.